src/compiler/asc.js

```diff
@@ -0,0 +1,71 @@
+const asc = require('assemblyscript/cli/asc')
+
+const createExitHandler = inputFile => () => {
+  throw new Error(`The AssemblyScript compiler crashed when compiling this file: '${inputFile}'
+Unfortunately this error has not been mapped yet.
```

Comment:
This error message is really nice and gentle! I'd remove this part:
> Unfortunately this error has not been mapped yet

It's not clear what it means and doesn't add much to the already long message.

Reply:
Ok, I'll do it, thanks!

Reply:
Fixed!